rules/integrations/azure/credential_access_azure_entra_suspicious_signin.toml

```
@@ -0,0 +1,100 @@
+[metadata]
+creation_date = "2025/04/28"
+integration = ["azure"]
+maturity = "production"
+updated_date = "2025/04/28"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies concurrent azure signin events for the same user and from multiple sources, and where one of the authentication
+event has some suspicious properties often associated to DeviceCode and OAuth phishing. Adversaries may steal Refresh
+Tokens (RTs) via phishing to bypass multi-factor authentication (MFA) and gain unauthorized access to Azure resources.
+"""
+false_positives = [
+    """
+    Users authenticating from multiple devices and using the deviceCode protocol or the Visual Studio Code client.
+    """,
+]
+from = "now-60m"
+language = "esql"
+license = "Elastic License v2"
+name = "Concurrent Azure SignIns with Suspicious Properties"
```

```suggestion
name = "Microsoft Entra ID Concurrent Sign-Ins with Suspicious Properties"
```

Just for consistency.